src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import url(_variables.scss);
+
+$field--background-color: #eee;
+$field--hover-color: pink;
```

Comment:
According previous comment you should move this variables to your file `_variables.scss` 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/a1e9c117-7bd3-481d-b40e-24dd4552b11d)